behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -400,12 +424,14 @@ public function assertLinkVisibleRegion($link, $region) {
   }
 
   /**
-   * @Then I should not see the link :link visible in the :region( region)
+   * @Then I should not visibly see the link :link in the :region( region)
```

Comment:
Yes you are right, I am following [Behat Drupal Extension](http://cgit.drupalcode.org/drupalextension/tree/src/Drupal/DrupalExtension/Context/MinkContext.php?id=857b164d18da5b0ab7208380f07c5f7620b198f7#n242). So in the future we don't have to remember which Context the steps were from.  =)